### DISCUSSION TITLE
Move the discussion to somewhere more convenient

### DISCUSSION THREAD

**Discussion by Kalrkloss** [CapibaraZero/fw#45] [General]
andreock, I don't know how you see this, but I've always considered the discussions in Github to be a pain in the back.
If you don't mind, I'd like to do it somewhere else, Discord maybe.
Only with your permission, of course.
If you don't have the time, I can open a Discord and give you administrative rights.

If not, could you please also allow discussions in the resources repository?

**Comment by andreock** [comment]
For the moment I prefer to maintain only Github discussion and issues so I can manage all the discussion on the same platform. In the feature i will consider the possibility of opening a Matrix server with a discord bridge maybe
